blog/zh/blog/2022/12/07/junrunrenli-with-apisix.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "APISIX 在君润人力云原生平台的架构实践"
+author: "袁鹏"
+keywords: 
+- 用户案例
+- 开源
+- APISIX
+- 君润人力
+description: 本文从君润人力业务快速扩张的背景入手，重点介绍开源 API 网关 APISIX 对其自研平台系统架构的多样化应用场景支持。
+tags: [Usercase]
+image: https://static.apiseven.com/2022/12/06/638ef9ee2c067.png
+---
+
+> 本文从君润人力业务快速扩张的背景入手，重点介绍开源 API 网关 APISIX 对其自研平台系统架构的多样化应用场景支持。
+
+<!--truncate-->
+
+> 分享嘉宾袁鹏，一页科技架构师
+
+君润人力成立于 2019 年，是一家以科技驱动的人力资源解决方案服务商，依托行业领先的科技水平和服务能力，专注于为客户提供一站式人力资源服务。自研数十家人力资源服务平台，深度链接 B 端企业和 C 端用户，构建数字化人力资源服务生态。
+
+本文从君润人力业务快速扩张的背景入手，重点介绍开源 API 网关 APISIX 对其自研平台系统架构的多样化应用场景支持，共有四大线上实战案例，希望对仍在网关选型过程中的企业或用户有所帮助。
+
+![君润人力业务](https://static.apiseven.com/2022/12/09/639291b9ce2a8.png)
+
+## 君润人力自研系统架构概述
+
+君润人力在搭建自研服务平台架构时，首要原则就是“开放、开源、拥抱云原生”。平台基于 K8s 微服务架构构建云端系统，整体架构参考下图。
+
+![自研服务平台](https://static.apiseven.com/2022/12/09/639291ba824c6.png)
+
+可以看到，君润自研服务平台 DevOps 流程（右侧）依托 Git webhook、coding、K8s 集群实现全自动化与无感发布，业务系统基于 PaaS 平台进行迭代开发，确保研发规范，并达成技术栈的统一。（上侧）在监控手段上，系统集成了 sky-agent 和 arthas，满足了线上服务观测多维度的需求。与此同时，采用腾讯日志云与 K8s 结合的方式将服务日志存储在云上，实现 K8s 集群内无文件化服务，避免磁盘容量堆积从而影响到 K8s 集群单个节点的健康。
```

Comment:
```suggestion
可以看到，君润自研服务平台 DevOps 流程（右侧）依托 Git webhook、CODING、K8s 集群实现全自动化与无感发布，业务系统基于 PaaS 平台进行迭代开发，确保研发规范，并达成技术栈的统一。（上侧）在监控手段上，系统集成了 sky-agent 和 arthas，满足了线上服务观测多维度的需求。与此同时，采用腾讯日志云与 K8s 结合的方式将服务日志存储在云上，实现 K8s 集群内无文件化服务，避免磁盘容量堆积从而影响到 K8s 集群单个节点的健康。
```

Does coding refer to Tencent Cloud's CODING? If so, you can consider committing the suggestion